Frontend/ui-driver/src/Screens/UploadDrivingLicenseScreen/Controller.purs

```diff
@@ -160,11 +160,18 @@ eval (CallBackImageUpload image imageName imagePath) state = if(state.props.clic
                                                           updateAndExit state {data {imageBack = image, imageNameBack = imageName}} $ ValidateImageAPICall $ state {data {imageBack = image, imageNameBack = imageName}}
                                                             else continue state
 
-eval (DatePicker (label) year month date) state = do
-  case label of
-    "DATE_OF_BIRTH" -> continue state {data = state.data { dob = (dateFormat year) <> "-" <> (dateFormat (month+1)) <> "-" <> (dateFormat date) <> " 00:00:00.233691+00" , dobView = (show date) <> "/" <> (show (month+1)) <> "/" <> (show year)}}
-    "DATE_OF_ISSUE" -> continue state {data = state.data { dateOfIssue = Just $ (dateFormat year) <> "-" <> (dateFormat (month+1)) <> "-" <> (dateFormat date) <> " 00:00:00.233691+00" , dateOfIssueView = (show date) <> "/" <> (show (month+1)) <> "/" <> (show year), imageFront = "null"}} -- imageFront made null to handle fallback
-    _ -> continue state
+eval (DatePicker (label) resp year month date) state = do
+  case resp of 
+    "SELECTED" -> case label of
+                    "DATE_OF_BIRTH" -> continue state {data = state.data { dob = (dateFormat year) <> "-" <> (dateFormat (month+1)) <> "-" <> (dateFormat date) <> " 00:00:00.233691+00" , dobView = (show date) <> "/" <> (show (month+1)) <> "/" <> (show year)}
+                                                        , props {isDateClickable = true }}
+                    "DATE_OF_ISSUE" -> continue state {data = state.data { dateOfIssue = Just $ (dateFormat year) <> "-" <> (dateFormat (month+1)) <> "-" <> (dateFormat date) <> " 00:00:00.233691+00" , dateOfIssueView = (show date) <> "/" <> (show (month+1)) <> "/" <> (show year), imageFront = "null"}
```

Comment:
you can store date in variable and can use same in both the cases like 
let date = (dateFormat year) <> "-" <> (dateFormat (month+1)) <> "-" <> (dateFormat date) <> " 00:00:00.233691+00"
    dateView = (show date) <> "/" <> (show (month+1)) <> "/" <> (show year)